src/client/pages/Home.jsx

```diff
@@ -1,13 +1,16 @@
 import React, { useContext } from 'react';
 import { Redirect } from 'react-router-dom';
-import usuarioContexto from '../state/state';
+import userContext from '../state/state';
```

Comment:
Here you can see the danger of exporting as default. Your variable is named AuthContext but you are importing as UserContext.